src/components/SmartWearableFilter/SmartWearableFilter.stories.css

```diff
@@ -0,0 +1,3 @@
+.dui-rarity-filter {
```

Comment:
This file might not be necessary, as there's no div wrapping the `SmartWearableFilter`, would you mind removing it?